### PR TITLE
Angela/logs reference content update - part 1

### PR DESCRIPTION
**Pull request by angelampcosta**

### CODE REVIEW

File: products/logs/src/content/reference/log-fields/index.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
 ---
-title: Datasets
+title: Log fields
```
Comment (kodster28) [inline products/logs/src/content/reference/log-fields/index.md:2]: If the title and the H1 are the same, you don't need this `title` field.

File: products/logs/src/content/reference/logpush-api-configuration/examples/example-logpush-curl/index.md [hunk 2]
```diff
@@ -65,15 +65,15 @@ https://api.cloudflare.com/client/v4/zones/<ZONE_ID>/logpush/jobs \
...
-* *destination_conf* - see [Destination](/reference/logpush-api-configuration/#destination) for details
-* *dataset* - the category of logs you want to receive; either  `http_requests` (default), `spectrum_events`, or `firewall_events`; cannot be changed after the job is created
-* *logpull_options* (optional) - see [Options](/reference/logpush-api-configuration/#options)
-    * Typically includes the desired fields and timestamp format
-    * Set the timestamp format to RFC 3339 (`&timestamps=rfc3339`) for:
-      * Google BigQuery usage
-      * Automated timestamp parsing within Sumo Logic; *see [timestamps from Sumo Logic](https://help.sumologic.com/03Send-Data/Sources/04Reference-Information-for-Sources/Timestamps%2C-Time-Zones%2C-Time-Ranges%2C-and-Date-Formats) for details*
-* *ownership_challenge* - challenge token required to prove destination ownership
+* **name** (optional) - We suggest using your domain name as the job name; the name cannot be changed after the job is created.
+* **destination_conf** - See [Destination](/reference/logpush-api-configuration/#destination) for details.
```
Comment (kodster28) [inline products/logs/src/content/reference/logpush-api-configuration/examples/example-logpush-curl/index.md]:
```suggestion
* **destination_conf** - Refer to [Destination](/reference/logpush-api-configuration/#destination) for details.
```

**Review by kodster28 — APPROVED** [review]

File: products/logs/src/content/get-started/enable-destinations/splunk/index.md [hunk 3]
```diff
@@ -61,17 +61,17 @@ Unlike configuring Logpush jobs for AWS S3, GCS, or Azure, there is no ownership
 
 To create a job, make a `POST` request to the Logpush jobs endpoint with the following fields:
 
-* `name` (optional) - Use your domain name as the job name.
-* `destination_conf` - A log destination consisting of an endpoint URL, channel id, insecure-skip-verify flag, sourcetype, authorization header in the string format below. 
+* **name** (optional) - Use your domain name as the job name.
+* **destination_conf** - A log destination consisting of an endpoint URL, channel id, insecure-skip-verify flag, source type, authorization header in the string format below.
```
Comment (soheiokamoto) [inline products/logs/src/content/get-started/enable-destinations/splunk/index.md:65]: Should it be like **sourcetype**? It is a specific parameter for Splunk.

Comment (angelampcosta) [reply products/logs/src/content/get-started/enable-destinations/splunk/index.md:65]: As Pedro previously mentioned, they're not consistent across their documentation. I think I would keep it as two words when we mention the parameter in the documentation, for instance "<SOURCE_TYPE>: The Splunk source type." But use the one word form for the examples, like in "destination_conf":"splunk://<SPLUNK_ENDPOINT_URL>?channel=<SPLUNK_CHANNEL_ID>&insecure-skip-verify=<INSECURE_SKIP_VERIFY>&sourcetype=<SOURCE_TYPE>&header_Authorization=<SPLUNK_AUTH_TOKEN>", "logpull_options": "fields=ClientIP,ClientRequestHost,ClientRequestMethod,ClientRequestURI,EdgeEndTimestamp,EdgeResponseBytes,EdgeResponseStatus,EdgeStartTimestamp,RayID&timestamps=rfc3339", "dataset": "http_requests"}' | jq .
But if you think it's better to always use a single word, we can change.

Comment (soheiokamoto) [reply products/logs/src/content/get-started/enable-destinations/splunk/index.md:65]: Sounds good.

**Pull request merged**